diagnostic.md

```diff
@@ -5,74 +5,85 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend does the heavy lifting.  It stores data, routes client requests and basically
+supports an application.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The Model, which talks to the database, crunches numbers and retrieves whatever the client needs from the database
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The Controller (it works with both the Model and the View)
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Because the view is what the user sees and interacts with (visual representation of the data they requested), so if we used them in our interpretation, I guess it would have just been someone standing there being poked or typed on?
```

Comment:
Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.